lib/core/src/Cardano/Pool/Metrics.hs

```diff
@@ -602,14 +599,13 @@ instance ToText StakePoolLayerMsg where
         MsgRegistry msg -> toText msg
         MsgListStakePoolsBegin -> "Listing stake pools"
         MsgMetadataUnavailable -> "Stake pool metadata is unavailable"
-        MsgComputedProgress (Just dbTip) nodeTip -> mconcat
+        MsgComputedProgress prodTip nodeTip -> mconcat
             [ "The node tip is:\n"
             , pretty nodeTip
             , ",\nbut the last pool production stored in the db"
```

Comment:
This is not _exactly_ true anymore. `prodTip` is now the greatest slot stored in the DB, where `epoch == nodeTip.epoch`. 

If the DB were to find itself at a greater slotId than the node this distinction would matter… But that shouldn't be possible.

edit: see other comment which is more concerning

Reply:
Well, any epoch stored that would be greater than the nodeTip is irrelevant. That the symptoms of a rollback that hasn't yet happened on the database. In practice, the most recent production tip from the node's epoch is our current state.